.gitignore

```diff
@@ -15,6 +15,7 @@ requests.egg-info/
 *.swp
 *.egg
 env/
+.idea/
```

Comment:
Please remove this. Pull requests should be singly purposed and not include unrelated changes.